apps/console/src/features/organizations/components/edit-organization-role/edit-organization-role.tsx

```diff
@@ -0,0 +1,159 @@
+/**
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { RoleConstants } from "@wso2is/core/constants";
+import { hasRequiredScopes } from "@wso2is/core/helpers";
+import { RolesInterface, SBACInterface } from "@wso2is/core/models";
+import { ResourceTab } from "@wso2is/react-components";
+import React, { FunctionComponent, ReactElement, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useSelector } from "react-redux";
+import { RoleGroupsList } from "./edit-organization-groups";
+import { RolePermissionDetails } from "./edit-organization-permission";
+import { BasicRoleDetails } from "./edit-organization-role-basic";
+import { RoleUserDetails } from "./edit-organization-role-users";
+import { AppState, FeatureConfigInterface, history } from "../../../core";
+import { OrganizationRoleInterface } from "../../models";
+
+/**
+ * Captures props needed for edit role component
```

Comment:
Let's add a full-stop for the comments. Check other places as well.

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/3617/commits/81488c55dd6c9c7a63575dee78e71f2b80643963